src/spryker/_twig/docker-compose.yml/environment.yml.twig

```diff
@@ -1,12 +0,0 @@
-{% for store, vhost in @('rabbitmq.vhosts') %}
```

Comment:
We need to keep this file or provide another way to have these environment variables present.

Reply:
Now resolved :balloon: